worker/buildbot_worker/runprocess.py

```diff
@@ -748,6 +780,11 @@ def cleanUp(self, hit):
             # blocks here until the process has terminated, while closing
             # stderr. This is weird.
             self.pp.transport.loseConnection()
+        elif runtime.platformType == 'win32':
+            if hasattr(self, 'job_object'):
```

Comment:
Better set job_object to None initially and then check for that, instead of using hasattr.

Reply:
Sorry, I totally forgot about this.